src/spaces/gradedspace.jl

```diff
@@ -200,7 +200,7 @@ end
 Base.summary(io::IO, V::GradedSpace) = print(io, type_repr(typeof(V)))
 
 function Base.show(io::IO, V::GradedSpace)
-    opn = (get(io, :typeinfo, Any)::DataType == typeof(V) ? "" : type_repr(typeof(V)))
+    opn = (get(io, :typeinfo, Any)::Type == typeof(V) ? "" : type_repr(typeof(V)))
```

Comment:
Is this type constraint really helpful? What is its specific purpose?

Reply:
Ok, might be worth checking if we have other patterns like this one that do not yet have the constraint, but maybe you did that already.